guide/glossary.md

```diff
@@ -0,0 +1,317 @@
+
+## Taproot
+
+A technique that makes complex multisig transactions look the same as standard transactions on the blockchain. This improves both efficiency and privacy, as multiple signatures are combined into a single one.
+
+## Transaction
+*ToDo*
+
+## Unspent transaction output (UTXO)
+An output that has not been sent to another address. The bitcoin wallet balance is calculated from adding up unspent outputs.
+
+##  Partially signed bitcoin transaction (PSBT)
+A file format for bitcoin transactions that are not fully signed yet. Allows for passing around a transaction to other applications or devices for signing, for example in a multi signature wallet setup.
+
+**References:**
+
+- [BIP174](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki){:target="_blank"}: Partially Signed Bitcoin Transaction Format
+
+## Vault
+
+A term sometimes used for multi-signature wallets.
+
+## Wallet
+
+<div class="center">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/wallet.jpg"
+   retina = "/assets/images/guide/glossary/wallet@2x.jpg"
+   alt-text = "Illustrations of different types of wallets."
+   layout = "float-right-desktop"
+   width = 400
+   height = 300
+%}
+
+This term is often used interchangeably for very different things. A user can *download a bitcoin wallet* in the app store, and then the app may offer the option to “create a new wallet” on the home screen. A software wallet may allow for browsing and sending transactions, while a hardware wallet may be limited to key storage and signing functionality.
+
+</div>
+
+### Bitcoin wallet
+
+All bitcoin-related data derived from and associated with a single recovery phrase. Most modern bitcoin wallets are HD wallets.
+
+### HD wallet
+
+Hierarchical Deterministic wallets, or HD wallets, can create infinite keypairs organized in a tree-structure (hierarchical) from a single (deterministic) controlling keypair. They were introduced and defined in [BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki){:target="_blank"} and then expanded with [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki){:target="_blank"} which added the ability to handle multiple *accounts* in one wallet. 
+
+**Technicalities** - There are different bitcoin address standards; Legacy, Native SegWit (segregated witness) and Nested SegWit. They have to be held in different branches of the HD wallet but can be controlled by the same recovery phrase. Native SegWit uses Bech32 address formats specified in [BIP173](https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki){:target="_blank"}.
```

Comment:
```suggestion
**Technicalities** - There are different bitcoin address formats (see [address](#address)). They have to be held in different branches of the HD wallet but can be controlled by the same recovery phrase.
```

Reply:
`They have to be held in different branches`. This is tricky to understand. What's a branch? Does the user have to manually do this? How about `There are different bitcoin address formats (see [address](#address)), which can be generated from the same recovery phrase, using different methods.`?

Reply:
> organized in a tree-structure

its tree branches, as stated in the paragraph above. i agree better analogies need to be better but "using different methods" brings more vagueness to the reader imo.